net/net.go

```diff
@@ -855,9 +854,6 @@ func (n *net) getLocalRecords(ctx context.Context, id thread.ID, lid peer.ID, of
 	if err != nil {
 		return nil, err
 	}
-	if lg.PubKey == nil {
-		return nil, fmt.Errorf("log not found")
-	}
```

Comment:
In what situation we could have a Log but without its public key?
I ask since this check is removed, but I don't see any other change regarding `GetLog` logic?
cc @sanderpick

Reply:
> In what situation we could have a Log but without its public key?

There wouldn't be a situation like that anymore since GetLog now returns an error if it's not found. This was the old way to check if it was "empty", since info is not a pointer receiver.

Reply:
Nice, much better then!